test/input/Elaborated.cxx

```diff
@@ -0,0 +1,49 @@
+namespace start {
+
+    // Struct
+    struct Foo1 {};
```

Comment:
The test output may be shorter if we use an incomplete type and reference it through pointer types.